composer.json

```diff
@@ -30,7 +30,10 @@
     "autoload-dev": {
         "psr-4": { "Aws\\Laravel\\Test\\": "tests/" }
     },
-    "extra": {
+  "config": {
+    "allow-plugins": true
```

Comment:
Why is this being set to true here?

Reply:
@SamRemis test suite breaks without it 

https://github.com/mateusjunges/aws-sdk-php-laravel/actions/runs/4177961439

Reply:
We should instead either allow only the required plugins, or ideally change the test file to use --no-plugins on the composer require statement that's causing that error to be thrown if that is an option.  

I'll spend some time tomorrow looking in to which plugin is causing this and if it's actually required for this test suite

Reply:
Hey @SamRemis I looked into this and indeed this plugin is not necessary for the test suite. I reverted https://github.com/aws/aws-sdk-php-laravel/pull/223/commits/943174f4ea223a18142d8de34920776d065cd196 and added `--no-plugins` when installing composer dependencies.